glide.lock

```diff
@@ -166,6 +166,12 @@ imports:
   version: e57e3eeb33f795204c1ca35f56c44f83227c6e66
 - name: github.com/spf13/viper
   version: c1de95864d73a5465492829d7cb2dd422b19ac96
+- name: github.com/stretchr/testify
+  version: 69483b4bd14f5845b5a1e55bca19e954e827f1d0
```

Comment:
before merging, if testify looks alright, i'll need to `glide install`. left them out of the review to reduce pr noise